rstudio/rstudio
# Fix overlocalized strings related to project settings

### Intent

Addresses #11054

### Approach

Don't localize strings that end up in the .Rproj file; another case of overlocalization. Put back the original strings, annotated with comment indicating these shouldn't be localized, and removed from the property files.

### Automated Tests

None

### QA Notes

Try repro steps in issue and confirm that the changes to project settings no longer result in localized words in the .Rproj file (and thus the project can be reopened without errors).

You can switch into French UI via the language selection option available in recent daily builds, under Global Options / General / Advanced (User Interface Language).

### Checklist
